session_security/templates/session_security/all.html

```
@@ -25,7 +25,7 @@
     {# Bootstrap a SessionSecurity instance as the sessionSecurity global variable #}
     {% with redirect_to_logout_flag=request|redirect_to_logout %}
     {% localize off %}
-        <script type="text/javascript"{% if request.csp_nonce %} nonce="{{ request.csp_nonce }}"{% endif %}>
+        <script type="text/javascript"{% if csp_nonce is not None %} nonce="{{ csp_nonce }}"{% elif request.csp_nonce %} nonce="{{ request.csp_nonce }}"{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)